web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -153,6 +181,34 @@
                           <div class="help-block with-errors"></div>
                       </div>
                   </c:when>
+
+                  <c:when test ="${data.stateName == 'tech'}">
```

Comment:
This section appears twice in this file, please double check and get rid off one of them if it is a duplicate.